docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -52,6 +52,7 @@ For more details on the wiring for your module, refer to the module's family pag
 pin. This is the PWM/DSHOT input for all Vertiq modules. In general, connection with your module will look as follows:
 
 .. image:: ../_static/tutorial_images/pwm_flight_controller/connect_with_module.png
+    :align: center
```

Comment:
Did this work? It looks left on mine for some reason
![image](https://github.com/user-attachments/assets/b4b3d590-3338-4ec2-a171-10bf5b2cd390)